### PR TITLE
Add missing blank

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: docs/level_2_2.html [hunk 1]
```diff
@@ -5,7 +5,7 @@ <h1 class="title topictitle1" id="ariaid-title1">Level 2</h1>
     <div class="body">
         <p class="p"><strong class="ph b">Drucktechnischen Gegebenheiten</strong> werden wiedergegeben und eine Interpretation von Zeichen
 orientiert sich am wenig Fällen am <strong class="ph b">Gebrauch im Sprach- und Schriftsystem</strong>. So werden zum
```
Comment (stweil) [inline docs/level_2_2.html:7]:
```suggestion
orientiert sich am <strong class="ph b">Gebrauch im Sprach- und Schriftsystem</strong>. So werden zum
```

**Comment by stweil** [comment]
@kba, @tboenig, which repository should get future pull requests, https://github.com/OCR-D/gt-guidelines or https://github.com/tboenig/Transkriptionsrichtlinien? A forced push makes tracking changes difficult on my side.

**Comment by kba** [comment]
Hi Stefan, sorry for the confusion. This is the repository to track. The force-push was unavoidable because the repository was so huge and, as you discovered one, there were a few variant repos too many. These should all be archived or deleted now.

We decided on keeping this repository but rebasing it to keep the issues and PRs such as yours.

@tboenig Can you check @stweil's changes and integrate them if they aren't yet?

**Comment by tboenig** [comment]
@stweil Hi Stefan, thank you very much for your corrections. I hope that I was able to fix some bugs for now.

**Pull request closed without merging**